src/test/java/org/jitsi/meet/test/web/WebTestBase.java

```diff
@@ -383,7 +383,7 @@ private void ensureInfoDialogClosed()
      * @param options the options to be used when creating the participant.
      * @return the participant which was created
      */
-    private WebParticipant joinParticipantAndWait(
+    public WebParticipant joinParticipantAndWait(
```

Comment:
I'm not a big fan of this change. What's the problem with using `joinConference` ?

Reply:
I may be wrong but it seems that 'ensureOneParticipant' method will do the same thing without exposing private method

Reply:
Oh you need an index there. The at least maybe make the method protected instead of public.

Reply:
Thanks @paweldomas. I talked to @damencho and he's not happy with this change either. He suggested to introduce a joinNextParticipant method and I'll try to implement that.